Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -298,3 +291,25 @@ extension Module  {
         return buildSettings
     }
 }
+
+extension Plist {
+    // FIXME: This is only internal for unit testing.
+    /// Quotes the strings in the Plist.
+    func quoteStrings() -> Plist {
+        switch self {
+        case .string(let str):
+            // Only quote if string has a space character.
+            guard str.utf8.contains(UInt8(ascii: " ")) else {
```

Comment:
Perhaps I'm misunderstanding the purpose of this function.  I was reacting to what seems like a problems in the code, which is that input strings containing, say, just a quote or a backslash end up broken.

Is this for build settings?  If so, then the appropriate distinction is probably to use strings vs arrays, and not to quote some strings and not others.

I'll take a look at the rest of the diffs to see if I can come up with more actionable suggestions.